ManageCoursesUi.sln.DotSettings.user

```diff
@@ -0,0 +1,4 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/Environment/AssemblyExplorer/XmlDocument/@EntryValue">&lt;AssemblyExplorer&gt;&#xD;
+  &lt;Assembly Path="C:\Users\cex\.nuget\packages\smartbreadcrumbs\1.0.1\lib\netcoreapp2.0\SmartBreadcrumbs.dll" /&gt;&#xD;
```

Comment:
This has your username in it. I think the *.user files should be ignored in the `.gitignore` file